iModelJsNodeAddon/api_package/ts/src/NativeLibrary.ts

```diff
@@ -649,6 +649,7 @@ export declare namespace IModelJsNative {
     public enableWalMode(yesNo?: boolean): void;
     public performCheckpoint(mode?: WalCheckpointMode): void;
     public setAutoCheckpointThreshold(frames: number): void;
+    public setSchemaECVersion(schemaName: string, ecVersion: ECVersion): boolean

```

Comment:
Is this method strictly for your test? If not, in what scenarios would core-backend call it?

Reply:
Sorry !
Premature commit, more changes incoming.